engines/python/setup/djl_python/rolling_batch/rolling_batch.py

```diff
@@ -336,9 +428,13 @@ def get_new_requests(self,
                 params = parameters[i] if i < len(parameters) else {}
                 adapter = adapters[i] if adapters is not None and i < len(
                     parameters) else None
+                details = params.pop("details", False)
```

Comment:
@lanking520 @sindhuvahinis this is where details are being parsed from the parameters the user sends

Reply:
Oh, got it.

Reply:
could we have this UX tested somewhere? https://github.com/deepjavalibrary/djl-serving/blob/master/engines/python/setup/djl_python/tests/test_test_model.py#L30 maybe here?

Reply:
Added test

Reply:
details experience doesn't change, it still can be passed from parameters